config/logstash.yml

```diff
@@ -159,7 +159,12 @@
 # Flag to turn on dead-letter queue.
 #
 # dead_letter_queue.enable: false
-#
+
+# If using dead_letter_queue.enable: true, the maximum size of each dead letter queue. Entries
+# will be dropped if they would increase the size of the dead letter queue beyond this setting.
+# Deafault is 1024mb
```

Comment:
@acchen97 this one is closed, but https://github.com/elastic/logstash/pull/7638#pullrequestreview-49410327 :)